### PR TITLE
remove unnecessary logs

### LINKED ISSUE #17546: remove logs
# [#17268] merged with some left over logs

### PR DESCRIPTION
**Pull request by jo-mut**
Simple pr. to remove some missed left over logs

fixes #17546

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

**Comment by jo-mut** [comment]
@qoqobolo am not sure this needs manual qa but if necessary then someone from QA can have a look. Thanks

**Review by OmarBasem — APPROVED** [review]

**Review by ajayesivan — APPROVED** [review]

**Comment by pavloburykh** [comment]
Hi @jo-mut could you please specify what do you mean by

> remove some missed left over logs

it is not clear out if this https://github.com/status-im/status-mobile/issues/17546 what exactly has been fixed and what should we test.

File: src/status_im2/common/bottom_sheet/view.cljs
Comment (jo-mut) [inline src/status_im2/common/bottom_sheet/view.cljs:77]: @pavloburykh  these are the logs I missed to clean up before merging the pr

**Comment by jo-mut** [comment]
@pavloburykh but then @OmarBasem realized a bug that had been introduced in my p.r afterwards and so i fixed it here. Looks like there is literary  no much to test just that the bottom margin of the bottom-sheet is right.

**Comment by jo-mut** [comment]
this is it https://github.com/status-im/status-mobile/commit/d5a71e2ce08579919ee3cf8d5f9b535a5d17b91b#r129206870

**Comment by pavloburykh** [comment]
ok, thank you for clarification. We will check and provide feedback

**Comment by pavloburykh** [comment]
@jo-mut is this issue supposed to be fixed by current PR or this is some separate problem? 

### ISSUE 1 Bottom sheets on Profiles list screen are detached from the footer (IOS)

Reproduced in IOS 16.7, iPhone X

**Steps:** 

1. Open Profiles list screen
2. Open "Remove profile" and "Create new profile" bottom sheets
3. Pay attention at the position of the bottom sheets

**Actual result:** 

photo_2023-10-06 12 58 56
photo_2023-10-06 12 59 00

**Expected result:** 

[Figma](https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?type=design&node-id=4357-629658&mode=design&t=Zn4jyKzFTWXD1GdR-0)

Onboarding for Mobile – Figma 2023-10-06 13-04-33
Onboarding for Mobile – Figma 2023-10-06 13-04-51

**Comment by jo-mut** [comment]
@pavloburykh ISSUE 1: is now fixed

**Comment by pavloburykh** [comment]
@jo-mut please take a look at the new issue

### ISSUE 2 All bottom sheets on Android have wrong bottom margin

Tested on Android 12, Samsung Galaxy A52

**Actual result:** margins are too small. 

photo_2023-10-11 15 14 34
photo_2023-10-11 15 14 38
photo_2023-10-11 15 14 42
photo_2023-10-11 15 14 46

**Comment by qoqobolo** [comment]
@jo-mut @smohamedjavid due to new [issues](https://github.com/status-im/status-mobile/pull/17547#issuecomment-1757567136), perhaps it will be more convenient if in the current PR will only remove the unnecessary logs, and the bottom margin will be fixed in https://github.com/status-im/status-mobile/pull/17609?

It would also be more convenient from a testing perspective - test all the fixes for bottom sheets at once and not wait for the merge of the current PR to include its changes.
Unless it's extra work for both of you, of course.
WDYT guys?

**Comment by jo-mut** [comment]
@smohamedjavid @qoqobolo  thats also okay

**Comment by jo-mut** [comment]
@qoqobolo I have updated the code to remove only the logs. I will add some comments on your pr @smohamedjavid for context

**Comment by qoqobolo** [comment]
Thank you @jo-mut!
@pavloburykh could you recheck the PR pls?

**Comment by pavloburykh** [comment]
> Thank you @jo-mut! @pavloburykh could you recheck the PR pls?

I have smoke checked. I think we can skip e2e step as only temporary js/console.log strings have been removed. Should not affect the app. Ready for merge. @jo-mut

**Pull request merged**
